src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -35,6 +36,10 @@ final class SlackTransport extends AbstractTransport
 
     public function __construct(string $accessToken, string $channel = null, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
     {
+        if (!\in_array(substr($accessToken, 0, 5), ['xoxb-', 'xoxp-'], true) && 0 !== strpos($accessToken, 'xoxa-2')) {
```

Comment:
I wonder if a regexp would'nt be faster in such case `/^xox(b-|p-|a-2)/`

Reply:
```suggestion
        if (!preg_match('/^xox(b-|p-|a-2)/', $accessToken)) {
```